webpack/assets/javascripts/react_app/components/ToastsList/ToastListHelpers.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-console */
+
+import { AlertVariant } from '@patternfly/react-core';
+
+import { translate as __ } from '../../common/I18n';
+
+export const toastType = type => {
+  if (type in AlertVariant) return type;
+
+  const message = `Toast notification type '${type}' is invalid. Please use one of the following types: ${Object.values(
+    AlertVariant
+  )}`;
+
+  switch (type) {
+    case 'alert':
+      console.warn(message);
+      return 'warning';
+    case 'notice':
+      console.warn(message);
+      return 'info';
+    case 'error':
+      console.warn(message);
+      return 'danger';
+    default:
+      console.error(message);
+      return 'default';
+  }
+};
+
+export const toastTitle = (title, type) => {
+  if (!title) {
+    const msg = __('Missing title for alert!');
+    console.warn(msg);
+  }
```

Comment:
IMHO we don't have to use `console.warn` here,
we can use the defaults for each type if the title is `undefined`